src/plugins/intel_cpu/tests/functional/single_layer_tests/concat.cpp

```diff
@@ -67,21 +79,30 @@ class ConcatLayerCPUTest : public testing::WithParamInterface<concatCPUTestParam
     void SetUp() override {
         targetDevice = CommonTestUtils::DEVICE_CPU;
 
-        int axis;
+        int64_t axis;
         std::vector<InputShape> inputShape;
         ElementType netPrecision;
         CPUSpecificParams cpuParams;
-        std::tie(axis, inputShape, netPrecision, cpuParams) = this->GetParam();
+        std::tie(axis, inputShape, netPrecision, configuration, cpuParams) = this->GetParam();
 
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
-        selectedType += std::string("_") + InferenceEngine::details::convertPrecision(netPrecision).name();
+        if (netPrecision == ElementType::i64 || netPrecision == ElementType::u64) {
+            auto i64Flag = configuration.find(InferenceEngine::PluginConfigInternalParams::KEY_CPU_NATIVE_I64);
+            if (i64Flag == configuration.end() || i64Flag->second == InferenceEngine::PluginConfigParams::NO) {
+                selectedType = makeSelectedTypeStr(selectedType, ElementType::i32);
+            } else {
+                selectedType = makeSelectedTypeStr(selectedType, ElementType::i64);
+            }
+        } else {
+            selectedType = makeSelectedTypeStr(selectedType, netPrecision);
+        }
```

Comment:
This logic occurs several times. Maybe we can create some helper that we can reuse in such places?